README.md

```diff
@@ -4,7 +4,7 @@
 
 UnJS is a collection of independent JavaScript libraries, tools, and utilities.
 
-Each project is usually made for a specific purpose in the ecosystem following the [Unix Philosophy](http://www.catb.org/~esr/writings/taoup/html/ch01s06.html) and maintained by its author and designated maintainers.
```

Comment:
Website works for me (only it is very slow to load, I guess HDD backed storage or something!)

There is also waybackmachine in case side really went down (I hope it doesnt') 

https://web.archive.org/web/20250428124748/http://www.catb.org/~esr/writings/taoup/html/ch01s06.html